src/style.css

```diff
@@ -2,0 +1,107 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding-top: 20px;
+}
+
+.search-bar__item {
+  display: flex;
+  align-items: center;
+
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+}
+
+.search-bar__item--big {
+  height: 70px;
+  line-height: 70px;
+
+  margin-bottom: 20px;
+  border-radius: 3px;
+}
+
+.search-bar__item--small {
+  height: 42px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, you've added a 'margin-bottom' which is fine, but ensure that you don't add 'margin-top' to the elements that follow.